dashboard/app/controllers/api/v1/amazon_future_engineer_controller.rb

```diff
@@ -29,7 +29,24 @@ def submit
       amazon_terms: afe_params['consentAFE'],
       new_code_account: current_user.created_at > 5.minutes.ago
     )
-  rescue Services::AFEEnrollment::Error => e
+
+    # If the teacher requested it, submit to CSTA as well
+    if to_bool(afe_params['csta'])
+      Services::CSTAEnrollment.new.submit(
+        first_name: afe_params['firstName'],
+        last_name: afe_params['lastName'],
+        email: afe_params['email'],
+        school_district_name: '',
+        school_name: '',
```

Comment:
We're not passing these from the client.  I'm intentionally leaving them unimplemented for now.  In the near future, I think we'll have to add a lookup using NCES id to get this information.